### DISCUSSION TITLE
Clarity around Supported Versions of NGINX

### DISCUSSION THREAD

**Discussion by RobertHorrox** [nginx/nginx#241] [Q&A] [answered]
Hello,

I am trying to find information around which versions of NGINX actively receive security updates.  I can find mostly third party sites listing versions of NGINX that are "End of Life" (see https://endoflife.date/nginx).  The only documentation from NGINX I can find is a blog post from 2020 on versioning (https://www.f5.com/company/blog/nginx/nginx-1-18-1-19-released).

I'm asking as it seems the kubernetes NGINX ingress controller is using an unsupported version of NGINX 1.25.5 (https://github.com/kubernetes/ingress-nginx?tab=readme-ov-file#supported-versions-table) and the InfoSec people I work with don't want us using EOL software.  Are the third party sites correct about the NGINX supported versions policy and if so can this be documented within the NGINX repository?

**Comment by viktorberezikov** [comment, marked as answer]
There are two versions of nginx - mainline and stable. Only the last odd version and the last even version are supported. All other versions are legacy. Current versions can be checked/downloaded here: https://nginx.org/en/download.html.
